Cargo.lock

```diff
@@ -4857,6 +4893,33 @@ dependencies = [
  "urlencoding",
 ]
 
+[[package]]
+name = "google-cloud-bigquery"
+version = "0.7.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c48abc8687f4c4cc143dd5bd3da5f1d7ef38334e4af5cef6de4c39295c6a3fd0"
+dependencies = [
+ "anyhow",
+ "arrow 50.0.0",
```

Comment:
It seems that the bigquery sdk depends on arrow. Can you have some investigation on how the arrow is used in the sdk? If it ultimately uses arrow to communicate with bigquery server, we may try converting the stream chunk directly to the arrow payload without encoding it to proto first.

Reply:
The interface we are relying on is a half-baked one, he only supports reading bigquery pb and converting it to arrow, writing is not supported, so we are relying on the lowest level sdk that only supports pb writing